docs/changelog.md

```diff
@@ -1,3 +1,34 @@
+## v2.6.0 [2024-10-24]
+
+### Added
+- [FOGL-8841]: Optional permission array property in configuration to manage config item access based on user role. [#437](https://github.com/fledge-iot/fledge-gui/pull/437)
+
+### Changed
+- [FOGL-8942]: Configuration items group tabs are sorted by the minimum order of config item in the group. [#471](https://github.com/fledge-iot/fledge-gui/pull/471)
+- [FOGL-8458]: Developer page access through developer sub-menu options. [#456](https://github.com/fledge-iot/fledge-gui/pull/456)
+ 
```

Comment:
remove this extra newline?

Reply:
> Developer page access through developer sub-menu options

.... along with expand/collapse sidebar functionality?